packages/beacon-node/src/node/nodejs.ts

```diff
@@ -197,6 +197,9 @@ export class BeaconNode {
     // TODO: Should this call be awaited?
     await db.pruneHotDb();
 
+    // Delete deprecated eth1 data to free up disk space for users
+    await db.deleteDeprecatedEth1Data();
```

Comment:
I don't think we need to do an `await` here? in that case adding some logs seems better in case we want to debug it